### PR TITLE
add later versions

### PR DESCRIPTION
**Pull request by mairaw**
Related to #16046

### CODE REVIEW

File: docs/core/porting/project-structure.md [hunk 1]
```diff
@@ -18,12 +18,12 @@ Learn how to create a solution that compiles for both .NET Framework and .NET Co
   - Allows you to write code for a specific .NET Framework version in your libraries through the use of compiler directives.
 
   *Unsupported scenarios:*
-  - Requires developers to use Visual Studio 2017 to open existing projects. To support older versions of Visual Studio, [keeping your project files in different folders](#support-vs) is a better option.
+  - Requires developers to use Visual Studio 2017 os later versions to open existing projects. To support older versions of Visual Studio, [keeping your project files in different folders](#support-vs) is a better option.
```
Comment (gewarren) [inline docs/core/porting/project-structure.md]:
```suggestion
  - Requires developers to use Visual Studio 2017 or a later version to open existing projects. To support older versions of Visual Studio, [keeping your project files in different folders](#support-vs) is a better option.
```

File: docs/core/porting/project-structure.md [hunk 2]
```diff
@@ -18,12 +18,12 @@ Learn how to create a solution that compiles for both .NET Framework and .NET Co
...
 
 - <a name="support-vs"></a>[**Keep existing projects and new .NET Core projects separate**](#keep-existing-projects-and-create-a-net-core-project)
 
   *What this is good for:*
-  - Continuing to support development on existing projects without having to upgrade for developers/contributors who may not have Visual Studio 2017.
+  - Continuing to support development on existing projects without having to upgrade for developers/contributors who may not have Visual Studio 2017 or later versions.
```
Comment (gewarren) [inline docs/core/porting/project-structure.md]:
```suggestion
  - Supporting development on existing projects for developers and contributors who may not have Visual Studio 2017 or a later version.
```

**Review by gewarren — COMMENTED** [review]
Left a couple suggestions. "Later versions" in the plural didn't sound quite right to me.

File: docs/core/porting/project-structure.md (hunk 1 above)
Comment (mairaw) [reply docs/core/porting/project-structure.md]: That's a good point. I'll open a PR later to fix all the occurrences of that in our docs then:
https://github.com/dotnet/docs/search?q=%22or+later+versions%22&unscoped_q=%22or+later+versions%22

**Review by gewarren — APPROVED** [review]

**Pull request merged**